snr.py

```diff
@@ -0,0 +1,139 @@
+import numpy as np
+import astropy.units as u
+
+
+def snr(counts, npix=1, nb=np.inf, background=0, darkcurrent=0,
+        readnoise=0, gain=0, ADerr=0.289):
+    """ A function to calculate the signal to noise ratio (SNR) of an
+    astronomical observation.
+
+    Parameters
+    ----------
+    counts (float or int)
+        Total number of counts in an arbitrary exposure time.
+    npix (int)
+        Number of pixels under consideration for the signal. Default is 1.
+    nb (int)
```

Comment:
I understand that this is a useful abbreviation but I might vote for `n_background` just to be a bit clearer.